src/main/java/io/wispforest/owo/command/debug/DamageCommand.java

```diff
@@ -19,26 +20,24 @@ public class DamageCommand {
 
     public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
         dispatcher.register(literal("damage")
+                .executes(context -> execute(context, DamageSource.OUT_OF_WORLD, context.getSource().getPlayerOrThrow(), 1))
+                .then(argument("source", DamageSourceArgumentType.damageSource())
+                        .executes(context -> execute(context, DamageSourceArgumentType.getDamageSource(context, "source"), context.getSource().getPlayerOrThrow(), 1))
+                        .then(argument("damage", FloatArgumentType.floatArg(0))
+                                .executes(context -> execute(context, DamageSourceArgumentType.getDamageSource(context, "source"), context.getSource().getPlayerOrThrow(), FloatArgumentType.getFloat(context, "damage")))))
```

Comment:
Also, extract these into method references and give them meaningful names. No lambdas in the command builder